apps/wallet-mobile/src/features/WalletManager/network-manager/network-manager.ts

```diff
@@ -100,7 +100,7 @@ export const networkConfigs: Readonly<Record<Chain.SupportedNetworks, Readonly<N
   [Chain.Network.Sancho]: {
     network: Chain.Network.Sancho,
     primaryTokenInfo: primaryTokenInfoAnyTestnet,
-    chainId: 0,
+    chainId: 450,
```

Comment:
hey @michaeljscript chainId is protocol level it should be 0 | 1 AFAIK, the previous network id was deprecated in favor of network, that is 'sancho' 'mainnet' and 'preprod' Im guessing this is what u r lf